osu.Game/Screens/Ranking/ResultsScreen.cs

```diff
@@ -164,15 +164,18 @@ private void load()
             {
                 buttons.Add(new RetryButton { Width = 300 });
```

Comment:
So I was a little surprised that this doesn't need its own treatment compared to `HotkeyRetryOverlay`, and as it turns out `ResultsScreen` has its own `allowRetry` parameter... It should probably be used instead of reading player's? Not 100% sure, it's a little messy.